server/routes/api/representatives.js

```diff
@@ -50,15 +50,32 @@ const ALLOWED_JURISDICTION_FILTERS = Object.keys(JURISDICTION_FILTER_MAP)
 // Endpoints
 
 // Get
-router.get('/:zipCode', async (req, res) => {
-  const { zipCode } = req.params
+router.get('/:searchText', async (req, res) => {
```

Comment:
This should be changed so the search address in not part of the route, but is provided as a request parameter in the body.

Reply:
I have updated this to have it as a URL query parameter.